src/processor/operator/persistent/reader/csv/base_csv_reader.cpp

```diff
@@ -205,8 +207,7 @@ bool BaseCSVReader::readBuffer(uint64_t* start) {
     auto readCount = fileInfo->readFile(buffer.get() + remaining, bufferReadSize);
     if (readCount == -1) {
         // LCOV_EXCL_START
-        throw CopyException(
-            stringFormat("Could not read from file {}: {}", fileInfo->path, posixErrMessage()));
+        handleCopyException(stringFormat("Could not read from file: {}", posixErrMessage()), true);
```

Comment:
I'm assuming for these filesystem-related errors we should throw regardless of whether have have `IGNORE_ERRORS` set

Reply:
I think so. @acquamarin and @mxwli ?